common/lib/util/utils.js

```diff
@@ -129,10 +129,16 @@ var Utils = (function() {
 	 * of another constructor
 	 * See node.js util.inherits
 	 */
-	Utils.inherits = (typeof(require) === 'function' && require('util') && require('util').inherits) || function(ctor, superCtor) {
+
+	Utils.inherits = function(ctor, superCtor) {
 		ctor.super_ = superCtor;
 		ctor.prototype = Utils.prototypicalClone(superCtor.prototype, { constructor: ctor });
 	};
+	try {
+        if (typeof(require) === 'function' && require('util') && require('util').inherits) {
+        	Utils.inherits = require('util').inherits;
+		}
+    } catch (err) {  /* Do Nothing. */ }
```

Comment:
Thanks. I think it's cleaner to support this by adding `inspects` and `inherits` keys to the `Platform` object, which will avoid the need to have a bunch of try-catches. WDYT?